modules/nf-core/lofreq/somatic/main.nf

```diff
@@ -12,15 +12,18 @@ process LOFREQ_SOMATIC {
     tuple val(meta3), path(fai)
 
     output:
-    tuple val(meta), path("*.vcf.gz"), emit: vcf
+    tuple val(meta), path("*somatic_final.snvs.vcf.gz"), emit: vcf_snvs
+    tuple val(meta), path("*somatic_final.snvs.vcf.gz.tbi"), emit: vcf_snvs_tbi
+    tuple val(meta), path("*somatic_final.indels.vcf.gz"), emit: vcf_indels
+    tuple val(meta), path("*somatic_final.indels.vcf.gz.tbi"), emit: vcf_indels_tbi
```

Comment:
Can you align the `,emit` statements please

Reply:
Sure, done.